src/test/groovy/net/rubygrapefruit/platform/file/FileEventFunctionsStressTest.groovy

```diff
@@ -192,7 +192,7 @@ class FileEventFunctionsStressTest extends AbstractFileEventFunctionsTest {
         def onslaught = new OnslaughtExecuter(
             (otherDirs.collect { otherDir ->
                 { ->
-                    Thread.sleep((long) (Math.random() * 100))
+                    Thread.sleep((long) (Math.random() * 100) + 100)
```

Comment:
This is to avoid the situation where we'd restart watching multiple times even before changes started trickling in, which is not what we want to watch here.